extension/src/test/suite/fileSystem/watcher.test.ts

```diff
@@ -27,7 +28,9 @@ suite('File System Watcher Test Suite', () => {
       expect(relativePattern.baseUri).to.deep.equal(
         getTestWorkspaceFolder().uri
       )
-      expect(relativePattern.pattern).equal(join('.git', '**'))
+      expect(relativePattern.pattern).equal(
+        joinWithForwardSlashes(['.git', '**'])
```

Comment:
[C] Maybe don't use the function here as a change to the function could break the required behaviour but the test will still pass.